src/main/java/uk/gov/ons/ctp/integration/rhsvc/service/impl/CaseServiceImpl.java

```diff
@@ -54,7 +52,7 @@
   @Autowired private ProductReference productReference;
   @Autowired private RateLimiterClient rateLimiterClient;
 
-  private MapperFacade mapper = new RHSvcBeanMapper();
+  private DateTimeFormatter dobFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
```

Comment:
might be nicer as static final , with appropriate naming convention  DOB_FORMATTER or somesuch?

Reply:
Changed